gcc/config/riscv/riscv.md

```diff
@@ -1138,7 +1138,7 @@
   [(set (match_operand:SUPERQI   0 "register_operand"     "=r,r")
 	(sign_extend:SUPERQI
 	    (match_operand:SHORT 1 "nonimmediate_operand" " r,m")))]
-  ""
+  "!TARGET_ZBB"
```

Comment:
It's standard pattern name so we still need that even TARGET_ZBB has a pattern in bitmanip.md.

You can add a define_expand pattern above to resolve such issue
```lisp
(define_insn_and_split "extend<SHORT:mode><SUPERQI:mode>2"
  [(set (match_operand:SUPERQI   0 "register_operand"     "")
	(sign_extend:SUPERQI
	    (match_operand:SHORT 1 "nonimmediate_operand" "")))]
```
and then add a `*` to the name of this pattern 
```lisp
(define_insn_and_split "*extend<SHORT:mode><SUPERQI:mode>2"
```

Reply:
Should be handled by the second commit